packages/quick_actions/test/quick_actions_test.dart

```diff
@@ -59,14 +61,30 @@ void main() {
     log.clear();
   });
 
-  test('runLaunchAction', () {
-    quickActions.runLaunchAction(null);
+  test('initialize', () async {
+    final Completer<bool> quickActionsHandler = Completer<bool>();
+    quickActions.initialize((String _) => quickActionsHandler.complete(true));
     expect(
       log,
       <Matcher>[
         isMethodCall('getLaunchAction', arguments: null),
       ],
     );
     log.clear();
+
+    await expectLater(await quickActionsHandler.future, isTrue);
```

Comment:
nit: I think it might be idiomatic to say `expect(quickActionsHandler.future, completion(isTrue))`